src/test/java/it/gov/pagopa/admissibility/event/processor/AdmissibilityProcessorConfigRankingTest.java

```diff
@@ -0,0 +1,372 @@
+
+        String jsonNotValid = "{\"initiativeId\":\"id_1\",invalidJson";
+        errorUseCases.add(Pair.of(
+                () -> jsonNotValid,
+                errorMessage -> checkErrorMessageHeaders(errorMessage, "[ADMISSIBILITY_ONBOARDING_REQUEST] Unexpected JSON", jsonNotValid)
+        ));
+
+        final String failingOnboardingChecksUserId = "FAILING_ONBOARDING_CHECKS";
+        String failingOnboardingChecks = TestUtils.jsonSerializer(
+                OnboardingDTOFaker.mockInstanceBuilder(errorUseCases.size(), initiativesNumber)
+                        .userId(failingOnboardingChecksUserId)
+                        .build()
+        );
+        errorUseCases.add(Pair.of(
+                () -> {
+                    Mockito.doThrow(new RuntimeException("DUMMYEXCEPTION")).when(onboardingCheckServiceSpy).check(Mockito.argThat(i->failingOnboardingChecksUserId.equals(i.getUserId())), Mockito.any());
+                    return failingOnboardingChecks;
+                },
+                errorMessage -> checkErrorMessageHeaders(errorMessage, "[ADMISSIBILITY_ONBOARDING_REQUEST] An error occurred handling onboarding request", failingOnboardingChecks)
+        ));
+
+        final String failingAuthorityData = "FAILING_AUTHORITY_DATA";
+        String failingAuthoritiesDataRetriever = TestUtils.jsonSerializer(
+                OnboardingDTOFaker.mockInstanceBuilder(errorUseCases.size(), initiativesNumber)
+                        .userId(failingAuthorityData)
+                        .build()
+        );
+        errorUseCases.add(Pair.of(
+                () -> {
+                    Mockito.doReturn(Mono.error(new RuntimeException("DUMMYEXCEPTION"))).when(authoritiesDataRetrieverServiceSpy).retrieve(Mockito.argThat(i->failingAuthorityData.equals(i.getUserId())), Mockito.any(), Mockito.any());
+                    return failingAuthoritiesDataRetriever;
+                },
+                errorMessage -> checkErrorMessageHeaders(errorMessage, "[ADMISSIBILITY_ONBOARDING_REQUEST] An error occurred handling onboarding request", failingAuthoritiesDataRetriever)
+        ));
+
+        final String failingRuleEngineUserId = "FAILING_RULE_ENGINE";
+        String failingRuleEngineUseCase = TestUtils.jsonSerializer(
+                OnboardingDTOFaker.mockInstanceBuilder(errorUseCases.size(), initiativesNumber)
+                        .userId(failingRuleEngineUserId)
+                        .build()
+        );
+        errorUseCases.add(Pair.of(
+                () -> {
+                    Mockito.doThrow(new RuntimeException("DUMMYEXCEPTION")).when(ruleEngineServiceSpy).applyRules(Mockito.argThat(i->failingRuleEngineUserId.equals(i.getUserId())), Mockito.any());
+                    return failingRuleEngineUseCase;
+                },
+                errorMessage -> checkErrorMessageHeaders(errorMessage, "[ADMISSIBILITY_ONBOARDING_REQUEST] An error occurred handling onboarding request", failingRuleEngineUseCase)
+        ));
```

Comment:
it's not necessary repeat these useCases, due to its presence in AdmissibilityProcessorConfigTest